CMakeLists.txt

```diff
@@ -63,8 +63,9 @@ find_package(LAPACK REQUIRED)
 find_package(PkgConfig)
 
 if (USE_MPI)
-  get_property(REQUIRED_MPI_COMPONENTS GLOBAL PROPERTY ENABLED_LANGUAGES)
-  list(REMOVE_ITEM REQUIRED_MPI_COMPONENTS CUDA)  # CUDA does not have a MPI component
+  # MPI-parallelization only happens with Fortran rather than ENABLED_LANGUAGES
```

Comment:
negative, for C-API/CXX-API examples we also need the C MPI, resp. the CXX (latter I'm not sure, might be we test always only use the C).
But we could easily depend on Fortran and C statically since all implementations seem to provide C anyway.

Reply:
Fortunately the code above is outdated. Sorry for this!  I found MPI is needed for both C and C++ in addition to Fortran.